Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/PurposeOfForDecider.java

```diff
@@ -104,9 +104,7 @@ private void logPurposeDecision(Boolean purposeFor, String hcid,
             purposeName = "PURPOSE OF";
         }
 
-        LOG.debug("PurposeOfForDecider decision for HCID: " + hcid
-            + " and Service Name: " + serviceName + " is = "
+        LOG.debug("PurposeOfForDecider decision for HCID: " + hcid + " and Service Name: " + serviceName + " is = "
```

Comment:
Optional: if you are creating another commit. Can we use the same logging convention using brackets replacing variable names?